src/server.h

```diff
@@ -1863,12 +1898,14 @@ struct redisServer {
     int repl_timeout;               /* Timeout after N seconds of master idle */
     client *master;     /* Client that is master for this slave */
     client *cached_master; /* Cached master to be reused for PSYNC. */
+    client *psync_master; /* Client used for psync during rdb load */
```

Comment:
is this term better? that's a sister variable for `master`, right? maybe move it above `cached_master`?
```suggestion
    client *master_2nd_chan; /* Client used for psync during rdb load */
```

Reply:
`master_2nd_chan` or `master_rdb_chan` may sound like this variable is used by the rdb channel, when it actually the main connection. I suggest `repl_provisional_master` / `repl_tmp_master` / `repl_in_progress_master`